tests/unit/test_cocip.py

```diff
@@ -929,7 +936,6 @@ def test_radiative_heating_effects_param(fl: Flight, met: MetDataset, rad: MetDa
         "dt_integration": np.timedelta64(10, "m"),
         "process_emissions": False,
         "humidity_scaling": ExponentialBoostLatitudeCorrectionHumidityScaling(),
-        "interpolation_bounds_error": True,
```

Comment:
Did we get rid of this parameter? I don't see where this is changed in the code elsewhere

Reply:
The parameter is still there, but there was some sort of bug in which an error wasn't raised despite contrails evolving out of bounds. It was somewhat tangled up with the scipy 1.9 implementation. The scipy 1.10 `RegularGridInterpolator` has changed a lot.